pkg/services/sqlstore/migrations/alert_mig.go

```diff
@@ -89,11 +89,14 @@ func addAlertMigrations(mg *Migrator) {
 			{Name: "node_id", Type: DB_NVarchar, Length: 255, Nullable: false},
 			{Name: "heartbeat", Type: DB_TimeStampz, Nullable: false},
 			{Name: "sequence", Type: DB_Int, Nullable: false},
+			{Name: "alertRunType", Type: DB_Varchar, Length: 50, Nullable: false},
 		},
 		Indices: []*Index{
 			{Cols: []string{"node_id", "heartbeat"}, Type: UniqueIndex},
+			{Cols: []string{"heartbeat", "sequence", "alertRunType"}, Type: UniqueIndex},
```

Comment:
Where would we use this index? if it is to lookup nodeId, do we need sequence in it?